.github/workflows/pr-stale.yml

```
@@ -0,0 +1,22 @@
+name: 🧼 Stale PR Check
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@8f717f0dfca33b78d3c933452e42558e4456c8e7
+        with:
+          days-before-close: -1
+          days-before-stale: 21
```

Maybe it should go stale after 7 days, and then closed after 14 days? Since stale PRs prevent other people from picking up the feature.

---

Some PRs are just waiting for review, and closing them might not seem fair. How about 14 and 14?
```suggestion
          days-before-close: 14
          days-before-stale: 14
```